a8c-files.php

```diff
@@ -230,8 +230,20 @@ function save_image_file( $override, $filename, $image, $mime_type, $post_id ) {
 	}
 
 	function get_upload_dir( $upload ) {
-		$upload['path'] = constant( 'LOCAL_UPLOADS' ) . $upload['subdir'];
-		$upload['basedir'] = constant( 'LOCAL_UPLOADS' );
+
+		// Maybe fix paths for multisite
+		if ( ! is_multisite() || ( is_main_network() && is_main_site() ) ) {
+			// If we're not on multisite, do nothing
+			$sites_path = '';
+		} elseif ( false !== stripos( $upload['subdir'], '/sites/' ) ) {
+			// If we maybe already have `/sites/` in the path, do nothing
+			$sites_path = '';
+		} else {
+			$sites_path = '/sites/' . get_current_blog_id();
+		}
```

Comment:
Perhaps this clause can be simplified as:
```
               $sites_path = '';
               if ( ( is_multisite() || ( ! is_main_network() && ! is_main_site() ) && 
                   ( false === stripos( $upload['subdir'], '/sites/' ) ) ) {
			$sites_path = '/sites/' . get_current_blog_id();
		}
```

Since we're not really doing anything with the first 2 cases

Reply:
Ok, I can check into this a bit more to make sure I understand it correctly.